integration_tests/src/main/python/window_function_test.py

```diff
@@ -57,17 +57,17 @@
 _grpkey_longs_with_decimals = [
     ('a', RepeatSeqGen(LongGen(nullable=False), length=20)),
     ('b', DecimalGen(precision=18, scale=3, nullable=False)),
-    ('c', DecimalGen(precision=18, scale=3))]
```

Comment:
Previously the key was a decimal but now it's a long, so are we losing test coverage with this change or are we not worried about that here?

Reply:
Should we make a UniqueIntegerGen and a UniqueDecimalGen too then?

Reply:
@jlowe, I did consider that lost coverage.  There's a couple of things working in favour of this change:
1. Note that in the tests that use these gens, the query orders by both `b` and `c`, both being `DecimalGen`.  We're only changing `c`, so the test still orders by `DECIMAL`. I don't think we're losing coverage.
2. The focus of the `negative_rows` test is on whether row offsets `< 0` work.  It isn't really affected much by the order-by column.

Reply:
@revans2, we could consider introducing a `UniqueDecimalGen`, but I don't think that's strictly necessary for this test.